lib/world.rb

```diff
@@ -10,6 +10,10 @@ def as_json
     @_wjson ||= JSON.parse(File.read(file), symbolize_names: true)
   end
 
+  def countries
+    as_json.map { |slug, _| country(slug) }
```

Comment:
I'd probably also have this guarantee to return the countries already sorted (by name). Most uses are probably going to want that, so may as well save them the work.